examples/dummy.py

```diff
@@ -35,3 +38,6 @@
     # All agents perform their actions
     observations, rewards, terminated, truncated, info = env.step(actions)
     done = any(terminated.values())
+
+replay = Replay.load("test")
+replay.play()
```

Comment:
Maybe call replay file `replay`, same as in `pettingzoo` example. Also update `example/replay.py` to load `replay`.